sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServices.java

```diff
@@ -111,6 +111,9 @@ JobStatistics dryRunQuery(
 
   /** An interface to get, create and delete Cloud BigQuery datasets and tables. */
   public interface DatasetService extends AutoCloseable {
+
+    static String TABLE_METADATA_BASIC_VIEW = "BASIC";
```

Comment:
I'd create an enum to represent this and pass the default one we have (STATS) explicitly.